content/en/docs/concepts/security/security-checklist.md

```diff
@@ -0,0 +1,153 @@
+If a cloud provider is used for hosting Kubernetes, the access from pods to the cloud
+metadata API `169.254.169.254` should also be restricted or blocked if not needed
+because it may leak information.
+
+## Pod security
+
+RBAC authorization is crucial but
+[cannot be specific enough on the Pods resource](/docs/concepts/security/rbac-good-practices/#workload-creation)
+(or on any resource that manages Pods). The only granularity is the API verbs
+on the resource itself, for example `create` on Pods. Indeed, without
+additional admission, the authorization to create these resources allows direct
+unrestricted access to on the schedulable nodes of a cluster.
+
+[Pod Security Admission](/docs/concepts/security/pod-security-admission/) is an
+admission controller to enable fine-grained authorization of Pod creation and
+updates. It replaces the Pod Security Policy admission controller. 
+The [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
+define three different policies that restrict how fields can be set in the `PodSpec`.
+For a hands-on tutorial on Pod Security, see the blog post
+"[Kubernetes 1.23: Pod Security Graduates to Beta](/blog/2021/12/09/pod-security-admission-beta/)".
+
+{{< note >}}
+[PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/)
+will be removed in version 1.25.
+{{< /note >}}
+Pod Security admission `restricted` policy can enforce or even warn of the part of the
+[security context](/docs/tasks/configure-pod-container/security-context/) that must
+be properly set. Nevertheless, pods' [security context](/docs/tasks/configure-pod-container/security-context/)
+should be separately investigated to limit the privileges and access pods may have.
+
+[Memory and CPU limits](/docs/concepts/configuration/manage-resources-containers/)
+should be set in order to restrict the memory and CPU resources a pod can consume on
+a node, and therefore prevent potential DoS attacks.
+
+## Enabling seccomp
+
+Seccomp can improve the security of your workloads by reducing the Linux kernel
+syscall attack surface available inside containers. The seccomp filter mode
+leverages BPF to create allow or deny list of specific syscalls, named
+profiles. Those seccomp profiles can be enabled on individual workloads,
+[a security tutorial is available](/docs/tutorials/security/seccomp/). In
+addition, the [Kubernetes Security Profiles Operator](https://github.com/kubernetes-sigs/security-profiles-operator)
+is a project to facilitate the management and use of seccomp in clusters.
+
+Please note that Docker has been using
+[a default seccomp profile](https://docs.docker.com/engine/security/seccomp/)
+to only allow a restricted set of syscalls but Kubernetes is still not
+confining workloads by default. Fortunately,
+[Seccomp Default](/blog/2021/08/25/seccomp-default/), a new alpha feature to
+use a default seccomp profile for all workloads can now be enabled and tested.
+
```

Comment:
## Enabling AppArmor

[AppArmor](https://apparmor.net/) can provide an easy way for Mandatory Access Control (MAC) and can provide better auditing through system logs. To [enable AppArmor in Kubernetes](https://kubernetes.io/docs/tutorials/security/apparmor/) at least Kubernetes version v1.4 is required. Like seccomp, AppArmor is also configured through profiles, where each profile is either running in enforcing mode, which blocks access to disallowed resources, or complain mode, which only reports violations. AppArmor profiles are defined per container, with an annotation, allowing for each pod to gain just the right privileges.